### PR TITLE
Bug 2103975: Add admin flag (for hiding Data Foundation navItem from non-admin users)

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
**Admin:**
Screenshot from 2022-08-04 15-08-46
**Non-Admin:**
Screenshot from 2022-08-04 15-05-43

### CODE REVIEW

**Review by bipuladh — CHANGES_REQUESTED** [review]
Please perform a SSAR check on the StorageSystem CR instead.

File: packages/odf/features.ts [hunk 1]
```diff
@@ -31,19 +35,16 @@ import {
 } from './constants';
 import { NooBaaSystemModel } from './models';
 
-export const ODF_MODEL_FLAG = 'ODF_MODEL';
-export const OCS_INDEPENDENT_FLAG = 'OCS_INDEPENDENT';
-export const OCS_CONVERGED_FLAG = 'OCS_CONVERGED';
-export const ODF_MANAGED_FLAG = 'ODF_MANAGED';
-export const LSO_FLAG = 'LSO';
```
Comment (SanjalKatiyar) [inline packages/odf/features.ts:38]: removed this flag as well...

**Comment by SanjalKatiyar** [comment]
Need to manually backport to 4.11, 4.10 and 4.9

**Pull request merged**